concent_api/signing_service_tests.ini

```diff
@@ -0,0 +1,10 @@
+[concent-api]
+API_URL = http://127.0.0.1:8000/api/v1/send/
+
+[middleman]
+HOST = 127.0.0.1
+PORT = 9054
+
+[ethereum-blockchain]
+GETH_ADDRESS = http://localhost:8545
+CONCENT_ETHEREUM_ADDRESS = 1FMKNGV5wRw39sxuHzFuKPnjMSibo98dWy
```

Comment:
Concent's ethereum address is `a5f3cd35385755e64b47f20a5d7334d7013f8015`

Reply:
:+1: done